code/modules/games/cards/lotto.dm

```diff
@@ -147,4 +161,4 @@
 	fire_fuel = 1
 
 /obj/item/toy/lotto_ticket/unprinted/attackby(obj/item/weapon/S, mob/user)
-	return 0
```

Comment:
You return that newline right the fuck now

Reply:
Seems like it gets pruned if you do it in the browser.